plugin/src/Internal/Cache.hs

```diff
@@ -248,13 +269,19 @@ emptyCache enable = do
     },
     finder,
     eps,
+    hug = Nothing,
+    moduleGraph = Nothing,
     baseSession = Nothing
   }
+emptyCache :: Bool -> IO (MVar Cache)
+emptyCache enable = do
+  emptyCacheWith newCacheFeatures {enable}
 
 initialize :: Cache -> IO Cache
 initialize cache = do
-  unless cache.initialized do
-    initUniqSupply 0 1
+  when False do
```

Comment:
???
We had better comment out this code with some note if we want to keep this code bit.

Reply:
oh, huh! I guess I tried if out if it works without this and forgot about it then!